app/decorators/article_decorator.rb

```diff
@@ -13,4 +13,10 @@ def author
   def sorted_comments
     object.comments.sorted_by_created_at
   end
+
+  def rating
+    return "Nobody has rated on this article yet" if object.ratings.blank?
```

Comment:
it's quick than `.blank?` in your case

```suggestion
    return "Nobody has rated on this article yet" unless object.ratings.any?
```